blocks/releases/releases.js

```diff
@@ -15,10 +15,12 @@ function mdToHTML(md) {
 const displayNames = {
   'helix-pipeline-service': 'Delivery Pipeline',
   'helix-admin': 'Admin API',
+  'helix-config': 'Config Service',
```

Comment:
shouldn't that be the repo name? i.e:

```suggestion
  'helix-config-service': 'Config Service',
```

Reply:
As discussed offline, let's skip `helix-config-service` updates for now and only add `aem-lib`.